agenta-backend/agenta_backend/services/app_manager.py

```diff
@@ -317,20 +302,21 @@ async def remove_app(app: AppDB):
     Arguments:
         app_name -- the app name to remove
     """
-    # checks if it is the last app variant using its image
```

Comment:
let's keep the comment

Reply:
Why? The variable name is already clear as it is.